rust/connlib/tunnel/src/client.rs

```diff
@@ -999,7 +999,7 @@ impl ClientState {
         let message = query.message;
 
         let Some(upstream) = self.dns_mapping.get_by_left(&query.local.ip()) else {
-            tracing::debug!("Received TCP packet for non-sentinel IP");
+            tracing::warn!("Received TCP packet for non-sentinel IP");
```

Comment:
Doesn't this happen if 1.1.1.1 is a resource and you query it directly with +tcp?

Reply:
We should only get here if the packet was originally for a sentinel server, because otherwise, we wouldn't have forwarded it to our TCP stack.

If 1.1.1.1 is a resource, the packet will bypass connlib DNS completely.

Reply:
Ah, thakns